internal/api/v2/types_gen.go

```diff
@@ -91,11 +96,18 @@ type TxResponse struct {
 }
 
 type UrlQuery struct {
-	Url   string `json:"url,omitempty" form:"url" query:"url" validate:"required,acc-url"`
+	Url string `json:"url,omitempty" form:"url" query:"url" validate:"required,acc-url"`
+}
+
+type QueryPagination struct {
 	Start uint64 `json:"start,omitempty" form:"start" query:"start"`
 	Count uint64 `json:"count,omitempty" form:"count" query:"count"`
 }
 
+type QueryOptions struct {
+	ExpandChains bool `json:"expandChains,omitempty" form:"expandChains" query:"expandChains"`
+}
```

Comment:
All changes to these types needs to be done via `types.yml`. This file is generated and should not be edited directly. LMK if you need help with `types.yml`.

Reply:
Done